src/kindhandlers/common/customObjectKindHandler.ts

```diff
@@ -32,8 +33,8 @@ function extractResourceVersion(resource: K8sResource, kindVersion: string, kind
   return {version, group};
 }
 
-function extractFormSchema(editorSchema: any) {
-  const schema: any = JSON.parse(JSON.stringify(editorSchema));
+export function extractFormSchema(editorSchema: any) {
+  const schema: any = cloneDeep(editorSchema);
```

Comment:
I would argue that cloning with JSON is faster based on some benchmarks I saw, but this is more readable

Reply:
interesting - i thought it would be the opposite! thanks!